test/functional/test_runner.py

```diff
@@ -223,6 +223,7 @@ def main():
                                      formatter_class=argparse.RawTextHelpFormatter)
     parser.add_argument('--combinedlogslen', '-c', type=int, default=0, metavar='n', help='On failure, print a log (of length n lines) to the console, combined from the test framework and all test nodes.')
     parser.add_argument('--coverage', action='store_true', help='generate a basic coverage report for the RPC interface')
+    parser.add_argument('--allow_uncovered_rpcs', action='store_true', help='Uncovered RPCs do not fail the test run')
```

Comment:
sort please :)

Reply:
It is sorted logically, comes right after --coverage